include/constants/event_objects.h

```diff
@@ -242,6 +242,7 @@
 #define OBJ_EVENT_GFX_HOOH                       238
 #define OBJ_EVENT_GFX_POKE_BALL                  239
 #define OBJ_EVENT_GFX_OW_MON                     240
+#define OBJ_EVENT_GFX_LIGHT_SPRITE               OBJ_EVENT_GFX_QUINTY_PLUMP
```

Comment:
Why is this Quinty Plump? That's a very opaque and unnecessary reference.

Reply:
A change from before expanded IDs. Assigning its own value now.